solidity/contracts/utility/ContractRegistry.sol

```diff
@@ -98,8 +98,16 @@ contract ContractRegistry is IContractRegistry, Owned, Utils {
             items[_contractName].isSet = false;
 
             // if there are multiple items in the registry, move the last element to the deleted element's position
+            // and modify last element's registryItem.nameIndex in items,
+            // to point to the right position in contractNames
             if (contractNames.length > 1)
-                contractNames[items[_contractName].nameIndex] = contractNames[contractNames.length - 1];
+                string lastContractNameString = contractNames[contractNames.length - 1];
+                uint unresgisteredIndex = items[_contractName].nameIndex;
```

Comment:
typo

Reply:
could you be more specific?
I don't see anything wrong.

Reply:
do you mean `unregisteredIndex` not ok?

Reply:
yes, sorry `unresgisteredIndex` should be `unregisteredIndex`

Reply:
@yudilevi thx, I have fixed it and commit again.